qa/suites/rbd/iscsi/base/install.yaml

```diff
@@ -9,6 +9,10 @@ tasks:
     - ceph orch host ls
     - ceph orch device ls
 - install:
-    extra_packages:
+    extra_system_packages:
+      deb:
+      - open-iscsi
+      - multipath-tools
+      rpm:
       - iscsi-initiator-utils
       - device-mapper-multipath
```

Comment:
device-mapper-multipath is the package which has `mpathconf` tool, which is explicitly mentioned as a requirement for `rpm`  and not for `deb` package manger systems, wondering if the same package exists on Ubuntu. (Excuse me, I don't have a ubuntu VM)